### PR TITLE
Add ability to define reasons for being in others and countdowns

### PR DESCRIPTION
**Pull request by torztomasz**
Resolves L2B-8502, L2B-8491, L2B-8494

### CODE REVIEW

File: packages/config/src/common/ScalingProjectDisplay.ts [hunk 1]
```diff
@@ -13,7 +14,8 @@ export type ScalingProjectDisplay = {
   slug: string
   /** Name of the category the scaling project belongs to */
   category: ScalingProjectCategory
-  isOther?: boolean
+  /** Reasons why the scaling project is not included in the other categories. If defined - project will be displayed as other */
```
Comment (maciekzygmunt) [inline packages/config/src/common/ScalingProjectDisplay.ts:17]:
```suggestion
  /** Reasons why the scaling project is included in the 'Other' category. If defined - project will be displayed as other */
```

File: packages/frontend/src/components/badge/badge.tsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ const badgeVariants = cva('rounded text-center font-medium', {
       brightYellow: 'bg-yellow-200 text-purple-700',
       pink: 'bg-pink-900 text-white',
       purple: 'bg-purple-100',
+      brand: 'bg-[#EFC0FF] text-brand',
```
Comment (maciekzygmunt) [inline packages/frontend/src/components/badge/badge.tsx:19]: use branded color, if we dont have it, add it to config

**Review by maciekzygmunt — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/badge/badge.tsx (hunk 2 above)
Comment (torztomasz) [reply packages/frontend/src/components/badge/badge.tsx:19]: this is some random color picked by Janek

File: packages/frontend/src/server/features/utils/is-in-past.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { UnixTime } from '@l2beat/shared-pure'
+
+export function isInPast(dateInSeconds: number) {
```
Comment (sz-piotr) [inline packages/frontend/src/server/features/utils/is-in-past.ts:3]: Feels weird. Should be inline in feature flags. Also `isInThePast`

File: packages/frontend/src/components/countdown.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        )}
+      >
+        {suffix}
+      </span>
+    </div>
+  )
+}
+
+function getTimeParts(timeLeft: number) {
+  const months = Math.floor(timeLeft / (60 * 60 * 24 * 30))
```
Comment (sz-piotr) [inline packages/frontend/src/components/countdown.tsx:100]: LOL

Comment (sz-piotr) [reply packages/frontend/src/components/countdown.tsx:100]: That's definitely wrong. Today is Dec 13th. If I see a countdown of 1 mo 2 days I expect Jan 15th. But december has 31 days so this countdown will actually expire at Jan 14th

File: packages/frontend/src/components/table/cells/project-name-cell.tsx [hunk 5]
```diff
@@ -79,6 +84,11 @@ export function ProjectNameCell({ project, className }: ProjectCellProps) {
         {project.data?.syncStatus?.isSynced === false && (
           <NotSyncedIcon syncedUntil={project.data?.syncStatus.syncedUntil} />
         )}
+        {env.NEXT_PUBLIC_FEATURE_FLAG_OTHER_PROJECTS &&
```
Comment (sz-piotr) [inline packages/frontend/src/components/table/cells/project-name-cell.tsx:87]: Same note about feature flags.

File: packages/frontend/src/components/countdown.tsx (hunk 4 above)
Comment (torztomasz) [reply packages/frontend/src/components/countdown.tsx:100]: Yeah, I know, forgot to do smth with it

File: packages/frontend/src/app/(top-nav)/scaling/projects/[slug]/page.tsx [hunk 6]
```diff
@@ -85,6 +87,13 @@ export default async function Page(props: Props) {
             />
           </div>
           <div className="w-full">
+            {env.NEXT_PUBLIC_FEATURE_FLAG_OTHER_PROJECTS &&
```
Comment (sz-piotr) [inline packages/frontend/src/app/(top-nav)/scaling/projects/[slug]/page.tsx:90]: featureFlags

File: packages/frontend/src/app/(top-nav)/scaling/projects/[slug]/_components/scaling-project-stats.tsx [hunk 7]
```diff
@@ -25,7 +26,10 @@ interface Props {
 
 export function ScalingProjectStats({ project, className }: Props) {
   const isOther =
-    env.NEXT_PUBLIC_FEATURE_FLAG_OTHER_PROJECTS && project.header.isOther
+    env.NEXT_PUBLIC_FEATURE_FLAG_OTHER_PROJECTS &&
```
Comment (sz-piotr) [inline packages/frontend/src/app/(top-nav)/scaling/projects/[slug]/_components/scaling-project-stats.tsx:29]: featureFlags

**Review by sz-piotr — CHANGES_REQUESTED** [review]
Left comments

File: packages/config/src/common/ReasonForBeingInOther.ts [hunk 8]
```diff
@@ -0,0 +1,38 @@
+export const REASON_FOR_BEING_OTHER = {
+  NO_PROOFS: {
```
Comment (sz-piotr) [inline packages/config/src/common/ReasonForBeingInOther.ts:2]: We also need UNSOUND_PROOFS. Doesn't have to be this pr

File: packages/frontend/src/server/features/scaling/summary/get-scaling-summary-entries.ts [hunk 9]
```diff
@@ -72,6 +72,7 @@ function getScalingSummaryEntry(
     provider: project.display.provider,
     dataAvailability,
     mainPermissions: project.display.mainPermissions,
+    reasonsForBeingOther: project.display.reasonsForBeingOther,
```
Comment (sz-piotr) [inline packages/frontend/src/server/features/scaling/summary/get-scaling-summary-entries.ts:75]: Shouldn't this be part of statuses?

**Review by sz-piotr — CHANGES_REQUESTED** [review]
Left comments

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**